kuma/users/views.py

```diff
@@ -745,6 +767,38 @@ def recover(request, uidb64=None, token=None):
     return render(request, 'users/recover_failed.html')
 
 
+@require_POST
+def create_stripe_subscription(request):
+    user = request.user
+
+    has_stripe_error = False
+    try:
+        customer = None
+        if not user.stripe_customer_id:
+            customer = stripe.Customer.create(
+                email=request.POST.get('stripe_email', ''),
+                source=request.POST.get('stripe_token', ''),
+            )
+            user.stripe_customer_id = customer.id
+            user.save()
+        else:
+            customer = stripe.Customer.retrieve(user.stripe_customer_id)
```

Comment:
It is now using the primary email of your account when signing you up for a subscription. I don't see how that would be a problem (since there's only one way to sign-up to stripe atm)

Reply:
I guess it's a bit paranoid but there are currently 40 users in our database that have an existing `stripe_customer_id` and they potentially set that up with a different email than they want to type in tomorrow. 

I would just feel more comfortable if the code was correct rather than lucky. 
Couldn't there be something like this:
```python
if customer:
    # Yay that old user.customer_strip_id 
    # But let's check it. 
    if customer.email != request.POST['stripe_email']:
        # No bueno, scrap that old customer record
        customer = ...
        user.strip_customer_id = customer.id
        user.save()
```

Reply:
I see, I've added the additional check as recommended. Thanks for the context!